samples/snippets/csharp/VS_Snippets_Remoting/System.Net.Dns/CS/dnsnewmethods.cs

```diff
@@ -22,6 +22,17 @@ public static void DoGetHostEntry(string hostname)
         }
 //</Snippet1>
 
+//<Snippet4>
+        public static void DoGetHostEntry(IPAddress address)
+        {
+            IPHostEntry host;
+
+            host = Dns.GetHostEntry(address);
```

Comment:
@svick This one and the avoidance of interpolated strings was because I found the samples in this state. I'll change it on this upcoming commit.